dream_layer_backend/img2img_workflow.py

```
@@ -226,7 +255,19 @@ def transform_to_img2img_workflow(data):
     if refiner_data['refiner_enabled']:
         logger.info("Injecting Refiner parameters...")
         workflow = inject_refiner_parameters(workflow, refiner_data)
-
+    
+    print(f"✅ Workflow transformation complete")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)